src/css/components/stat.scss

```diff
@@ -13,77 +13,78 @@ $input-font-size: $sans-s2;
   display: flex;
   justify-content: space-between;
   min-height: 2.5rem; // Leave some space for input element when editing a stat
+  position: relative;
   width: 100%;
 
   .error_message {
     font-size: $input-font-size;
   }
 }
 
+h5 + .stat {
```

Comment:
Is this rule still used? I think we're getting rid of all the `h5`s in this PR https://github.com/18F/cg-dashboard/pull/855